lib/internal/bridge/devices_mixin.dart

```diff
@@ -0,0 +1,30 @@
+part of internal_bridge_lib;
+
+mixin DevicesMixin on FlutterBLE {
+
+
+  Future<List<Peripheral>> knownDevices(List<String> peripheralIdentifiers) async {
+    return _methodChannel.invokeMethod(MethodName.knownDevices, <String, dynamic>{
+      ArgumentName.deviceIdentifiers: peripheralIdentifiers,
+    })
+    .then((peripheralsJson) {
+      print("known devices json: $peripheralsJson");
+      return _parsePeripheralsJson(peripheralsJson);
+    });
+  }
+
+  Future<List<Peripheral>> connectedDevices(List<String> serviceUUIDs) async {
+    return _methodChannel.invokeMethod(MethodName.connectedDevices, <String, dynamic>{
+      ArgumentName.serviceUuids: serviceUUIDs,
+    })
+    .then((peripheralsJson) {
+      print("connected devices json: $peripheralsJson");
```

Comment:
We should log this only if the specified log level allows it

Reply:
https://github.com/Polidea/FlutterBleLib/issues/145